pom.xml

```diff
@@ -104,6 +104,10 @@
 			<version>${jjwt.version}</version>
 			<scope>runtime</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-docker-compose</artifactId>
+		</dependency>
```

Comment:
I am not sure we need this dependency because we do not work with compose programmatically

Reply:
removed